general/administration/installing.md

```diff
@@ -116,15 +129,49 @@ Windows builds in ZIP archive format are available [here](https://repo.jellyfin.
 1. Rename `system-bak` to `system`
 1. Run `jellyfin.bat` to start the server again
 
-### MacOS
+### Install using legacy script
+
+Only for versions 10.3.x and earlier.
+
+1. Download and extract the latest version
+1. Run `install.bat` or `install-jellyfin.ps1`
+1. (optional) Enter a custom install location (default is `%appdata%`)
+1. (optional) Enter a custom library/data location (default is `%localappdata%`) - Currently not working
+1. (optional) Select to install jellyfin as a service
+1. (optional) Import old library - Not compatible with newer Emby versions
+1. (optional) Create a shortcut on your desktop
+1. (optional) Run Jellyfin after installation
+1. Press `Install`
```

Comment:
I feel that it's awkward when we say "when using installer please uninstall zip first" and at the same time we don't provide uninstall instructions for zip...

Reply:
Didn't I write that in the note?

Reply:
I don't see any instructions on _how_ to uninstall stuff which was installed as zip